sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ServersTable.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  DataFilters,
+  DataSummary,
+  Toolbar,
+  DataSearch,
+  Data,
+  DataTable,
+  Pagination,
+} from 'grommet';
+import {
+  StatusCriticalSmall,
+  Upload,
+  SettingsOption,
+  StatusGoodSmall,
+  StatusPlaceholderSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
+import opsRamp from '../../../../mockData/opsRamp.json';
+
+import { QuickFilters } from './QuickFilters';
+
+const defaultView = {
+  search: '',
+  sort: { property: 'name', direction: 'asc' as 'asc' | 'desc' },
+  step: 10,
+  properties: {},
+};
+
+interface ServersTableProps {
```

Comment:
We should probably rename anything referring to "servers" as "nodes".

A Kubernetes cluster is comprised of a collection of nodes. High-level there is a master node and a bunch of worker nodes.